pkg/controller/handler.go

```diff
@@ -235,6 +273,23 @@ func (h *handler) CreateServiceBroker(in *servicecatalog.Broker) (*servicecatalo
 	return h.apiClient.Brokers().Update(in)
 }
 
+func (h *handler) unbind(b *servicecatalog.Binding) error {
+	inst, err := instanceForBinding(h.apiClient, b)
+	if err != nil {
+		return err
+	}
+	sc, err := serviceClassForInstance(h.apiClient, inst)
+	if err != nil {
+		return nil
+	}
+	broker, err := brokerForServiceClass(h.apiClient, sc)
```

Comment:
@MHBauer for now, yes, this is an API we should put on the client. we can reconsider when we add a cache to the controller, however